core/src/main/java/com/isbx/androidtools/media/ImageResizer.java

```diff
@@ -204,7 +206,14 @@ public Uri scaleImage(Uri sourceUri, ImageResizeConfig.Dimension targetDimension
 
                 ExifInterface exif = new ExifInterface();
                 exif.readExif(context.getContentResolver().openInputStream(sourceUri));
-                exif.writeExif(out, os);
+
+                if (exif.getAllTags() != null) {
+                    exif.writeExif(out, os);
+                } else {
+                    ExifInterface exif2 = new ExifInterface();
+                    exif2.addDateTimeStampTag(exif.TAG_DATE_TIME_ORIGINAL, Calendar.getInstance().getTime().getTime(), TimeZone.getDefault());
```

Comment:
@ebbnormal One last thing, static fields should be referenced by class, not by instance